tests/run-make/jobserver-error/not_a_pipe.stderr

```diff
@@ -0,0 +1,2 @@
+error: failed to connect jobserver from `MAKEFLAGS=Some("--jobserver-auth=3,3")`: file descriptor 3 from the jobserver environment variable value is not a pipe
```

Comment:
Why does it have to be a pipe? Ends of a unix socket should would fine too for that purpose, no?

Reply:
POSIX jobserver should be a pipe.

https://github.com/rust-lang/jobserver-rs/pull/52#discussion_r1135263485